HW Progress Section (#7)
teachee-capstone/teachee: reports/blueprint/report.tex

\end{titlepage}
\section*{Executive Summary}
% pop a one pager in here. Ideally less
% Timothy
\newpage

\tableofcontents
\listoffigures
\listoftables
\newpage
\setstretch{1.5}
\section{Introduction} \label{sec:intro} % Ethan
This blueprint outlines Group 18's capstone project TeachEE, which was

> originally proposed in ELEC390 \cite{prop_390}. This report is intended the ELEC 498 instructors

Reviewer: ```suggestion
originally proposed in ELEC390 \cite{prop_390}. This report is intended for the ELEC 498 instructors
```

> and Group 18's supervisors. The report provides the baseline requirements for

Reviewer: Maybe better to say specs
```suggestion
and Group 18's supervisors. The report provides the baseline specifications for
```

> TeachEE. These requirements should be treated as the minimum performance

Reviewer: ```suggestion
TeachEE. These specifications should be treated as the minimum performance
```

> required for the showcase of the project in March. This blueprint consists of a
> work breakdown structure, progress up to the time of writing this document,

Reviewer: ```suggestion
required for the showcase of the project in March. This blueprint consists of the methodology of completing the project, progress up to the time of writing this document,
```

budget, and strategies to mitigate issues. Appendices are

> included for relevant technical information that does not fit in the main body

Reviewer: Better to say it includes info that supplements the report

Author: Fixed

given by the target percentages in the hardware specifications.

\section{Progress to Date} % Ethan / Tim for HW/SW respectively
Since TeachEE is a project originally proposed in ELEC390, there is progress in

> the development of hardware and software.

Reviewer: What is this supposed to mean. Perhaps better to just say something like "since the start of the term, the team has accomplished a few hardware and software development tasks"

Author: I was just afraid of using past tense. will fix

they take care of both the fabrication and soldering of the PCBs. Moreover,
Bittele takes the generated output from the EDA software and procures all the
parts directly to their facility.

\begin{table}[h!]
    \caption{TeachEE Bill of Materials}
    \begin{tabularx}{\textwidth}{l|l|l}
        \textbf{Item / Part Number} & \textbf{Supplier} & \textbf{Cost (USD)} \\
        \hline
        PCB Fabrication & Bittele & \$104.37\\
        PCB Total BOM Procurement & Bittele & \$126.67\\
        PCB Assembly & Bittele & \$289.76\\
        \hline
        Total & & \$520.80\\
        \hline
        Budget Remaining & & \$79.20
    \end{tabularx} 
\label{tab:abbreviated-bom}
\end{table}

\noindent
As shown above, a full unit of the PCB comes in \$80 under budget. This leaves
the time with some additional budget to purchase any replacement components or
cover unexpected expenses. Should the project still exceed the budget, the team
will first contact the project supervisors to see if there is a possibility of
additional funding. The fully expanded component BOM for the PCB is provided in
Appendix \ref{appendix:bom}.

> 

Reviewer: Isn't there supposed to be two subsections here? Split this into materials/supplies and contributions from other sources

Reviewer: Ignore
